### PR TITLE
Fix up metadata/doc items to get started with the CW310 FPGA

### PR DESCRIPTION
**Pull request by a-will**
As I went through the "Getting Started on FPGAs" user guide, I encountered a couple bumps while using cw310_loader.py. These two little updates should smooth that out for the next user.

### CODE REVIEW

File: python-requirements.txt [hunk 1]
```diff
@@ -29,6 +29,7 @@ pyserial
 pygments
 pytest
 pytest-timeout
+pyusb >= 1.1.0
```
Comment (imphil) [inline python-requirements.txt:32]: Is this a dependency used by the cw310 library? In that case, the dependency should go into the chipwhisperer/cw310 library (here probably: https://github.com/newaetech/chipwhisperer/blob/develop/setup.py#L17)

**Review by imphil — COMMENTED** [review]
Thanks @a-will for your pull request! Documentation PRs are always highly appreciated. 

I have one question for the python-requirements.txt patch, can you have a look at that?

File: python-requirements.txt (hunk 1 above)
Comment (a-will) [reply python-requirements.txt:32]: Yes, and thanks for pointing out the repo this should go in. We can remove that commit from this series, then.

Comment (a-will) [reply python-requirements.txt]: It looks like the latest library has moved on from pyusb to libusb1. At least chipwhisperer will spit out an informative warning about what's missing, so I'd guess we're okay with skipping this. This commit is removed, so the series is now just the little flag to configure the PLL.

**Review by tjaychen — APPROVED** [review]

**Review by moidx — APPROVED** [review]

**Pull request merged**